api/web_components/index.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: default
+title: Web Components
```

Comment:
let's keep these unpublished until we have an API key requirement.

Reply:
Switched to `published: false`.